accern_xyme/accern_xyme.py

```diff
@@ -751,6 +753,46 @@ def get_settings(self) -> Dict[str, Any]:
         assert self._settings is not None
         return self._settings
 
+    def get_timing(
+                self,
+                blacklist: Optional[List[str]] = None,
+                ) -> Optional[TimingResult]:
+        blacklist = [] if blacklist is None else blacklist
```

Comment:
call the resulting list differently
`blist = [] if blacklist is None else blacklist`
and use that in the function -- this way you won't get type issues